cloud/pkg/nodeupgradejobcontroller/controller/downstream.go

```diff
@@ -210,7 +210,7 @@ func (dc *DownstreamController) processUpgrade(node string, upgrade *v1alpha1.No
 		State:    v1alpha1.Upgrading,
 		History: v1alpha1.History{
 			HistoryID:   upgradeReq.HistoryID,
-			UpgradeTime: time.Now().String(),
+			UpgradeTime: time.Now().Format(ISO8601UTC),
```

Comment:
How about using `RFC3339`, which is a golang const format? @WillardHu

Reply:
```golang
package time

const (
    ....
    RFC3339     = "2006-01-02T15:04:05Z07:00"
    ....
)
```

RFC3339 has a numeric time zone offsets(07:00), can we use `metav1.Time.MarshalJSON` to format `updateTime`?
```golang
tt := metav1.Time{Time: time.Now()}
bff, err := tt.MarshalJSON()
if err != nil {
    // handler error
}
string(bff)  // 2023-10-12T03:57:14
```

Reply:
Seems there's have no difference woth `ISO8601UTC` above, I think it's also fine to use `ISO8601UTC` :)